app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :rents, dependent: :destroy
```

Comment:
Be careful with `dependent: :destroy`. In this case, you may want to keep the rents although the user no longer exists.
Maybe `:nullify`?